proposals/0153-compensate-for-the-inconsistency-of-nscopyings-behaviour.md

```diff
@@ -224,4 +220,8 @@ The proposal doesn't change the ABI of existing language features.
 
 ## Alternatives Considered
 
-There is no alternatives considered at this time.
```

Comment:
Declaration in swift-evolution: <https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170213/032318.html>